src/integration/pushResult.js

```diff
@@ -45,17 +45,21 @@ function addCommit(src, msg){
 //pulling from remote repo
```

Comment:
Then add `--quiet` flag to git so it won't show anything unless there is an error